src/languages/erlang.js

```diff
@@ -82,32 +82,25 @@ export default function(hljs) {
   };
   const TRIPLE_QUOTE = {
     scope: 'string',
-    match: /"""("*)(?!")(.|\n)*?"""\1/,
+    match: /"""("*)(?!")[\s\S]*?"""\1/,
```

Comment:
What's the nuance here?

Reply:
I was just looking at the code for the BACKSLASH_ESCAPE and I thought it made a bit more sense than `(.|\n)`.